packages/core/src/command/hl7-sftp-ingestion/alohr/hl7-alohr-sftp-ingestion.ts

```diff
@@ -0,0 +1,20 @@
+import { log as createLog } from "../../../util/log";
+
+export const HIE_NAME = "Alohr";
+
+export const log = createLog("HL7-ALOHR-SFTP-INGESTION");
```

Comment:
i think we usually use lowercase kebab (hl7-alohr-sftp-ingestion) for log prefixes? not sure, but lets stick w whatever the pattern is.